services/movie.py

```diff
@@ -1,11 +1,13 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
```

Comment:
The type hint `list[int]` is used here, which is correct for Python 3.9 and later. Ensure that your project is using a compatible Python version.